src/main/java/uk/gov/pay/api/model/Payment.java

```diff
@@ -1,33 +1,37 @@
 package uk.gov.pay.api.model;
 
-public class Payment implements PaymentJSON {
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+public abstract class Payment {
+
+    public static final String LINKS_JSON_ATTRIBUTE = "_links";
+
+    @JsonProperty("payment_id")
```

Comment:
I am not a big fan of using JsonProperties on attributes that do not need them, but I understand that we might want them there for code consistency.